firstvoices/backend/views/join_request_views.py

```diff
@@ -135,7 +135,9 @@ class JoinRequestViewSet(
 
     def get_queryset(self):
         site = self.get_validated_site()
-        return JoinRequest.objects.filter(site__slug=site[0].slug).select_related(
+        return JoinRequest.objects.filter(
+            site__slug=site[0].slug, status=JoinRequestStatus.PENDING
+        ).select_related(
             "site", "site__language", "created_by", "last_modified_by", "user"
```

Comment:
Minor: could you please update the api doc for the list to say "pending join requests" instead of just "join requests"?